tests/unit/config/test_configfiles.py

```diff
@@ -617,6 +617,18 @@ def test_fonts_tabs(self, yaml, autoconfig):
         assert data['fonts.tabs.unselected']['global'] == val
         assert data['fonts.tabs.selected']['global'] == val
 
+    def test_content_media_capture(self, yaml, autoconfig):
+        val = 'ask'
+        autoconfig.write({'content.media_capture': {'global': val}})
+
+        yaml.load()
+        yaml._save()
+
+        data = autoconfig.read()
+        assert data['content.media.audio_capture']['global'] == val
+        assert data['content.media.audio_video_capture']['global'] == val
+        assert data['content.media.video_capture']['global'] == val
```

Comment:
Could probably make this a bit nicer with a for-loop:

```python
        for setting in ['content.media.audio_capture',
                        'content.media.audio_video_capture',
                        'content.media.video_capture']:
            assert data[setting]['global'] == val
```

Reply:
Again I don't really speak python, but that looks good to me. I'll commit it

Reply:
Done!